gapis/api/vulkan/draw_call_mesh.go

```diff
@@ -42,6 +44,15 @@ func drawCallMesh(ctx context.Context, dc *VkQueueSubmit, p *path.Mesh, r *path.
 
 	c := getStateObject(s)
 
+	cmd, err := resolve.Cmd(ctx, cmdPath, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if !cmd.CmdFlags().IsExecutedDraw() || len(cmdPath.Indices) == 1 {
+		return nil, &service.ErrDataUnavailable{Reason: messages.ErrMessage("Please select a draw call")}
+	}
```

Comment:
This is the wrong error (there is an existing "not a draw call" error) to return and probably the wrong place as well. We should not even get here for anything other than a draw call.
There is an issue with how meshes were implemented in Vulkan compared to GLES, which is now hard to see, as the GLES code is gone. There is a loop in the resolve that tries each node in the requested range that expects certain behavior of things below it. These expectations are currently not met, which is why we are seeing issues.